<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgImportVector</class>
 <widget class="QDialog" name="DlgImportVector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import vector layer</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Schema:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cboSchema"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Table:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="cboTable">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="radCreate">
        <property name="text">
         <string>Create new table</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkDropTable">
          <property name="text">
           <string>Drop existing one</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radAppend">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Append data into table</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkPrimaryKey">
        <property name="text">
         <string>Primary key:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editPrimaryKey"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkGeomColumn">
        <property name="text">
         <string>Geometry column:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editGeomColumn"/>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="chkSourceSrid">
          <property name="text">
           <string>Source SRID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editSourceSrid"/>
        </item>
        <item>
         <widget class="QCheckBox" name="chkTargetSrid">
          <property name="text">
           <string>Target SRID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editTargetSrid"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkEncoding">
        <property name="text">
         <string>Encoding:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cboEncoding">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="chkSinglePart">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Create single-part geometries instead of multi-part</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="chkSpatialIndex">
        <property name="text">
         <string>Create spatial index</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboSchema</tabstop>
  <tabstop>cboTable</tabstop>
  <tabstop>radCreate</tabstop>
  <tabstop>chkDropTable</tabstop>
  <tabstop>radAppend</tabstop>
  <tabstop>chkSourceSrid</tabstop>
  <tabstop>editSourceSrid</tabstop>
  <tabstop>chkEncoding</tabstop>
  <tabstop>cboEncoding</tabstop>
  <tabstop>chkSinglePart</tabstop>
  <tabstop>chkSpatialIndex</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgImportVector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
